src/LayerPlan.cpp

```diff
@@ -458,10 +458,10 @@ void LayerPlan::addPolygonsByOptimizer(const Polygons& polygons, const GCodePath
         addPolygon(polygons[poly_idx], orderOptimizer.polyStart[poly_idx], config, wall_overlap_computation, wall_0_wipe_dist, spiralize, flow_ratio, always_retract);
     }
 }
-void LayerPlan::addLinesByOptimizer(const Polygons& polygons, const GCodePathConfig& config, SpaceFillType space_fill_type, int wipe_dist, float flow_ratio, std::optional<Point> near_start_location)
+void LayerPlan::addLinesByOptimizer(const Polygons& polygons, const GCodePathConfig& config, SpaceFillType space_fill_type, bool enable_travel_optimization, int wipe_dist, float flow_ratio, std::optional<Point> near_start_location)
 {
     Polygons boundary;
-    if (config.type == PrintFeatureType::Infill && comb_boundary_inside.size() > 0)
```

Comment:
@smartavionics I suppose we should still check the line type because that flag is meant for infill?

Reply:
That test is now redundant because whether to use the travel optimization is now controlled by the new boolean arg to addLinesByOptimizer() and that arg will only be true when it is called for infill. But in the future we may want to use the optimization for lines that aren't infill (support maybe?) so it's better not to require the line type to be infill - that's my view.

Reply:
OK, makes sense.